src/components/operator/OperateButton.tsx

```diff
@@ -0,0 +1,19 @@
+interface OperateButtonProps {
+  operate: string;
+  onOperatorClick: (operator: string) => void;
+}
+
+export default function OperateButton({
+  operate,
+  onOperatorClick,
+}: OperateButtonProps) {
+  const operatorButtonClickHandler = () => {
+    onOperatorClick(operate);
+  };
+
+  return (
+    <button className="operation" onClick={operatorButtonClickHandler}>
```

Comment:
```suggestion
  return (
    <button className="operation" onClick={() => onOperatorClick(operate)}>
```

operatorButtonClickHandler에서 하는 일이 onOperatorClick 함수를 호출해 주고 operate를 넘겨주는 것밖에 없어서 위와 같이 간소화 해도 되지 않을까요?!